lib/talkable/configuration.rb

```diff
@@ -1,19 +1,21 @@
 module Talkable
   class Configuration
-    DEFAULT_SERVER = "https://www.talkable.com".freeze
+    DEFAULT_SERVER  = 'https://www.talkable.com'.freeze
+    DEFAULT_TIMEOUT = 5
 
     attr_accessor :site_slug
     attr_accessor :api_key
     attr_accessor :server
+    attr_accessor :read_timeout
+    attr_accessor :open_timeout
+    attr_accessor :timeout
     attr_accessor :js_integration_library
 
     class UnknownOptionError < StandardError
     end
 
     def initialize
-      self.site_slug  = ENV["TALKABLE_SITE_SLUG"]
-      self.api_key    = ENV["TALKABLE_API_KEY"]
-      self.server     = DEFAULT_SERVER
+      default_configuration
```

Comment:
how about to make `default_configuration` be a hash and always use `apply(...)` instead of explicitly setting an attribute aka `self.something = ...`

Reply:
What the problem with explicitly setting the attributes? Very clear